iroha/src/kura.rs

```diff
@@ -38,20 +38,22 @@ impl Kura {
         Ok(())
     }
 
-    /// Methods consumes new validated block and atomically stores and caches it.
-    pub async fn store(
-        &mut self,
+    pub fn create_block(
```

Comment:
Such a method should be inside `Block`'s impl block.

Reply:
Yes, thanks, I was also thinking about it